### ISSUE TITLE
Problem is not there @ 1KHz polling rate

### ISSUE THREAD

**Issue by kalligator** [DedLigma/a4tech-bloody-mouse-fix#1]
I just bumped into your [thread](https://bbs.archlinux.org/viewtopic.php?id=286436) today after sadly facing the same problem. 

Since it's been more than a year you may have discovered it already but setting the mouse refresh rate to 1KHz from the vendor's Windows app makes the problem go away.

My take is the mouse driver not communicating it's working refresh rate and Linux assigns the default 1KHz polling rate. So when set @ 2k it misbehaves due to the mismatch, when set @ 1k it functions properly.

What refresh rate does [evhz](https://aur.archlinux.org/packages/evhz-git/) give you with the fix? Is there a difference without?

**Comment by DedLigma** [comment]
Hello! A lot of time has passed since that problem... I can't tell you the exact refresh rate of my mouse because I changed my mouse a couple of months ago. I was using an A4tech Bloody J90 and I don't remember if I changed the refresh rate and it started working correctly. The only information about that problem at the time was in [this article](https://benyaamin.com/post/mouse-movement-issue-on-linux/). Now I found the new [solution](https://benyaamin.com/post/a-better-approach-to-handling-mouse-movement-issues-on-linux/) from the same person. Maybe you're right and the solution is much simple and setting the correct fresh rate solves the problem. However, I don't recall the refresh rate having a significant impact on the mouse's on Linux
